repair/repair.cc

```diff
@@ -1042,14 +1025,19 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
         // may be set, except data_centers may contain only local DC (-local)
         if (options.data_centers.size() == 1 &&
             options.data_centers[0] == topology.get_datacenter()) {
-            ranges = get_primary_ranges_within_dc(db.local(), keyspace);
+            // get_primary_ranges_within_dc() is similar to get_primary_ranges(),
+            // but instead of each range being assigned just one primary owner
+            // across the entire cluster, here each range is assigned a primary
+            // owner in each of the clusters.
```

Comment:
typo: in each of the DCs.

Reply:
yeah, I see it was preexisting and I merely moved the comment here, but I'll fix it while at it.

Reply:
fixed in v3